src/layouts/BlogPostLayout.astro

```diff
@@ -42,6 +42,18 @@ import "../styles/global.css";
           <h1 class="text-3xl">{frontmatter.title}</h1>
         </header>
         <slot />
+        <style is:global>
+          h3, p {
+             padding-bottom: 10px;
+          }
```

Comment:
I would just put this inline using the tailwind utilities. It would keep spacing more proportional. 

There should also be a "gap" property that might be more appropriate here.

Reply:
Not sure how to use TW for the blog content, its all in the `<slot />` tag. True gap might be better

Reply:
sweet